src/Resources/views/CRUD/base_list.html.twig

```diff
@@ -79,6 +79,9 @@ file that was distributed with this source code.
                                             {% endif %}
 
                                             {% spaceless %}
+                                                {% if field_description.options.header_style is defined %}
+                                                    {% deprecated 'The "header_style" is deprecated, please, use "header_class" instead.' %}
```

Comment:
Did you check if this works? Since it was merged after the last tagged release

Reply:
Also in the docs, it says from 2.6 and latest version is 2.5

Reply:
Oh yes, you are totally right! Moved check to php code